enterprise_access/apps/api/v1/views.py

```diff
@@ -746,7 +746,128 @@ def partial_update(self, request, *args, **kwargs):
         return response
 
 
-class SubsidyAccessPolicyViewset(PermissionRequiredMixin, viewsets.GenericViewSet):
+class SubsidyAccessPolicyCRUDViewset(PermissionRequiredMixin, viewsets.ModelViewSet):
+    """
+     Viewset for Subsidy Acess Policy CRUD operations.
+     """
+
+    permission_classes = (permissions.IsAuthenticated,)
+    serializer_class = serializers.SubsidyAccessPolicyCRUDSerializer
+    authentication_classes = (JwtAuthentication,)
+    filter_backends = (filters.OrderingFilter, DjangoFilterBackend,)
+    filterset_fields = ('group_uuid', 'policy_type',)
+    pagination_class = PaginationWithPageCount
+    http_method_names = ['get', 'head', 'options', 'post', 'patch', 'delete']
```

Comment:
Any idea what is the default value of this? Do we really need to add all the actions?

Reply:
Good catch, I will check if removing this line would have any effect on the listed actions.

Reply:
These are all the methods that are allowed by default https://github.com/django/django/blob/16411b8400ad08f90c236bb2e18f65c655f903f8/django/views/generic/base.py#L34